src/AutoCrudAdmin/Extensions/TypeExtensions.cs

```diff
@@ -43,6 +43,19 @@ public static IEnumerable<PropertyInfo> GetPrimaryKeyPropertyInfos(this Type typ
             .Where(property => primaryKeyNames != null && primaryKeyNames.Contains(property.Name));
     }
 
+    /// <summary>
+    /// The GetForeignKeyPropertyInfos method retrieves the foreign key property information for a given type.
+    /// </summary>
+    /// <param name="type">The Type instance that this method extends.</param>
+    /// <returns>An IEnumerable of PropertyInfo instances representing the foreign key properties of the given type.</returns>
+    public static IEnumerable<PropertyInfo> GetForeignKeyPropertyInfos(this Type type)
+        => ReflectionHelper.DbContexts
+            .Select(CreateDbContext)
+            .Where(dbContext => dbContext != null)
+            .SelectMany(dbContext => dbContext!.Model.FindEntityType(type) !.GetForeignKeys())
```

Comment:
Can you make it more safe, without using `!` before `GetForeignKeys()`. Ensure you get foreign keys only on entity types that can be found